clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -241,8 +241,6 @@ public DaVinciBackend(
           clusterInfoProvider,
           storeRepository,
           schemaRepository,
-          Optional.empty(),
-          Optional.empty(),
```

Comment:
Nice! I didn't expect the cleanup to reach all the way here, but that's good! It means the code is better decoupled...